test/reasoning.spec.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-env jest */
+
+const fs = require('fs');
+const { db } = require('../lib');
+const {
+  seedDatabase,
+  dropDatabase,
+  generateDatabaseName,
+  ConnectionFactory,
+} = require('./setup-database');
+
+const { reasoning, transaction } = db;
+
+describe('reasoning commands', () => {
+  const database = generateDatabaseName();
+  const conn = ConnectionFactory();
+
+  const beginTx = transaction.begin.bind(null, conn, database);
+  const commitTx = transaction.commit.bind(null, conn, database);
+
+  beforeAll(seedDatabase(database));
+  afterAll(dropDatabase(database));
+
+  it('should be able to check consistency', () => {
+    return reasoning.consistency(conn, database, {}).then(res => {
+      expect(res.status).toBe(200);
+      expect(res.body).toEqual('true');
+    });
+  });
+
+  it('should explain inferences', () => {
+    return reasoning
+      .explainInference(conn, database, '<urn:A> a <urn:B> .', {
+        contentType: 'text/turtle',
+      })
+      .then(res => {
+        expect(res.status).toBe(200);
+        expect(res.body.proofs).toBeTruthy();
```

Comment:
It asserts that the response body came back with a "proofs" key. I don't particularly care what the value is